docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3059,10 +3059,13 @@ what parts of the entity specification make up the *key* for each P4 entity.
 
 An `update` can be one of the following types:
 
-* `INSERT`: Inserts the given P4 entity in the entity container. If the entity
-  already exists, an `ALREADY_EXISTS` error is returned, and the existing entity
-  remains unchanged. The `entity` field always specifies the full state of the
-  P4 entity. If entity is malformed, an `INVALID_ARGUMENT` error is returned.
+* `INSERT`: Inserts the given P4 entity in the entity container.
+  The `entity` field always specifies the full state of the P4 entity.
+  If the entity already exists, an `ALREADY_EXISTS` error is returned, and
+  the existing entity remains unchanged.
+  If the entity is malformed, an `INVALID_ARGUMENT` error is returned.
+  If the entity cannot be inserted because the table is already full,
```

Comment:
use "container" here instead of "table" to be consistent with the beginning of the paragraph. Table is too specific I think.

Reply:
Done.